README.md

```diff
@@ -8,7 +8,7 @@ The TSC will be responsible for all aspects of oversight relating to the Project
 * Coordinating the technical direction of the Project;
 * Approving project or system proposals (including, but not limited to, incubation,
   deprecation, and changes to a sub-project’s scope);
-* Organising sub-projects and removing sub-projects;
+* Organizing sub-projects and removing sub-projects;
```

Comment:
You're correcting Her Majesty the Queen's English here I have you know! 

just kidding ;-)